webextensions/api/extension.json

```
@@ -344,7 +344,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/extension/onRequest",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "4"
```

:x: https://github.com/chromium/chromium/commit/5e34bc403af2328e80765406eace2f49f63e7d79 in [4.0.229.1](https://github.com/chromium/chromium/releases/tag/4.0.229.1) AFTER 4.0.223.

```suggestion
                "version_added": "5"
```